src/app/export/export.page.html

```diff
@@ -0,0 +1,21 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-title>Export your data</ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content>
+  <p class="ion-padding-start ion-padding-end">
+    Choose your carpools to export:
```

Comment:
Currently the Language is in German, pls use German.
Dual-language Supports will follow